ext/IntegralsZygoteExt.jl

```diff
@@ -44,14 +54,21 @@ function ChainRulesCore.rrule(::typeof(Integrals.__solvebp), cache, alg, senseal
             if sensealg.vjp isa Integrals.ZygoteVJP
                 if cache.batch > 0
                     dfdp = function (x, p)
-                        _, back = Zygote.pullback(p -> cache.f(x, p), p)
+                        z, back = Zygote.pullback(p -> cache.f(x, p), p)
+                        # messy, there are 4 cases, some better in forward mode than reverse
+                        # 1: length(y) == 1 and length(p) == 1
+                        # 2: length(y) >  1 and length(p) == 1
+                        # 3: length(y) == 1 and length(p) >  1
+                        # 4: length(y) >  1 and length(p) >  1
```

Comment:
I'm not sure what this comment is here for? I mean, agreed these are the 4 cases and sometimes forward is better than reverse, but I don't understand why that's here 😅